src/doc/en/developer/walkthrough.rst

```diff
@@ -62,35 +48,35 @@ So first, open a shell (for instance, Terminal on Mac) and check that
     concept guides. See 'git help <command>' or 'git help <concept>'
     to read about a specific subcommand or concept.
 
-
-Don't worry about the giant list of subcommands. You really only need
-a handful for effective development, and we will walk you through them
-in this guide. If you got a "command not found" error, then you don't
-have git installed. Now is the time to install it; see
+Don't worry about the giant list of subcommands. You really only need a handful
+of them for effective development, and we will walk you through them in this
+guide. If you got a "command not found" error, then you don't have Git
+installed; now is the time to install it. See
 :ref:`chapter-git-setup` for instructions.
 
-Because we also track who does changes in Sage with git, you must tell
-git how you want to be known. This only needs to be done once::
-
-    [user@localhost]$ git config --global user.name "Your Name"
-    [user@localhost]$ git config --global user.email you@yourdomain.example.com
+Because we also track who does what changes with Git, you must tell
+Git how you want to be known. Check if Git knows you::
 
-If you have multiple accounts / computers use the same name on each of
-them. This name/email combination ends up in commits, so do it now
-before you forget!
+    [alice@localhost ~]$ git config --global user.name
+    Alice Adventure
+    [alice@localhost ~]$ git config --global user.email
+    alice@wonderland.com
 
+If you have multiple computers, then use the same name on each of them.  This
+name/email combination ends up in commits. So if it's not set yet, do it now
+before you forget! This only needs to be done once. See
+:ref:`section-git-setup-name` for instructions.
 
 .. _section-walkthrough-sage-source:
 
-Obtaining and Compiling the Sage Source Code
-============================================
+Obtaining the Sage Source Code
+==============================
 
-Obviously one needs the Sage source code to develop.  You can use your
+Obviously one needs the Sage source code to develop. You can use your
 local installation of Sage, or (to start from scratch) download it
-from GitHub which is a public read-only mirror (=faster) of our
-internal git repository::
+from our Sage repository on GitHub::
 
-    [user@localhost ~]$ git clone https://github.com/sagemath/sage.git
+    [alice@localhost ~]$ git clone https://github.com/sagemath/sage.git
     Cloning into 'sage'...
```

Comment:
I think this should use `--origin upstream` so that the remote is labeled consistently.

Reply:
Yes. Nice. Thanks.